### PR TITLE
Fix/times of day dashboard properties,and height error

### PR DESCRIPTION
**Pull request by LumbardhAgaj**
Use backward compatible period values for times of day widget.
Remove height property from times of day widget scatter chart.

### CODE REVIEW

File: plugins/times-of-day/frontend/public/templates/times-of-day-widget.html [hunk 1]
```diff
@@ -13,6 +13,6 @@
     <clyd-primary-legend :apps="data.apps" :show-period="false"></clyd-primary-legend>
 
     <div class="clyd-widget__content">
-        <times-of-day-scatter-chart :series="series" :maxSeriesValue="maxSeriesValue" :show-zoom="false" :show-toggle="false" :showDownload="false" height="auto" skin="full"></times-of-day-scatter-chart>
+        <times-of-day-scatter-chart :series="series" :maxSeriesValue="maxSeriesValue" :show-zoom="false" :show-toggle="false" :showDownload="false" skin="full"></times-of-day-scatter-chart>
```
Comment (itsiprikshit) [inline plugins/times-of-day/frontend/public/templates/times-of-day-widget.html:16]: why did you remove height: auto?

Comment (LumbardhAgaj) [reply plugins/times-of-day/frontend/public/templates/times-of-day-widget.html:16]: It seems like we are stretching the charts to parent height only. I will revert it.
